.github/workflows/release.yml

```diff
@@ -37,7 +37,9 @@ jobs:
         run: dotnet pack --configuration Release /p:NuGetVersion=${VERSION} /p:PackageVersion=${VERSION} --include-symbols --output .
 
       - name: Push
-        run: dotnet nuget push FSharp.Data.Validation.${VERSION}.nupkg --source https://api.nuget.org/v3/index.json --api-key ${MTA_NUGET_KEY} --symbol-source https://symbols.nuget.org/download/symbols --symbol-api-key ${MTA_NUGET_KEY} 
+        run: |
+          dotnet nuget push FSharp.Data.Validation.${VERSION}.nupkg --source https://api.nuget.org/v3/index.json --api-key ${MTA_NUGET_KEY} --symbol-source https://symbols.nuget.org/download/symbols --symbol-api-key ${MTA_NUGET_KEY}
+          dotnet nuget push FSharp.Data.Validation.Async.${VERSION}.nupkg --source https://api.nuget.org/v3/index.json --api-key ${MTA_NUGET_KEY} --symbol-source https://symbols.nuget.org/download/symbols --symbol-api-key ${MTA_NUGET_KEY}
```

Comment:
@devinlyons Does it make sense to release this in lock-step with FSharp.Data.Validation? I opted to do it this way since it directly references FSharp.Data.Validation.

Reply:
That direct reference will be converted to a NuGet reference. However, the version range will be locked down to the exact version. I guess that's a long way of saying yes, let's release them in lock step.